lua/caching_handlers.lua

```diff
@@ -180,7 +181,7 @@ function caching_handlers.caching_proxy(incoming_zipkin_headers)
     end
 
     local handler_response
-    local handler_fn = cacheability_info.bulk_support
+    local handler_fn = cacheability_info.cache_entry['bulk_support']
```

Comment:
Keep using the same "dot" notation (`cache_entry.bulk_support`) for consistency

Reply:
Good catch!